Improve documentation about the cast operator. (#6261)
MicrosoftDocs/PowerShell-Docs: reference/7.1/Microsoft.PowerShell.Core/About/about_Operators.md

converted, PowerShell generates an error.

```powershell
[DateTime]"2/20/88" - [DateTime]"1/20/88"
[Int] (7/2)
[String] 1 + 0
[Int] '1' + 0
```

> Note: A cast will also be performed when a variable is assigned to using [cast notation](about_Variables.md).

Reviewer: ```suggestion
A cast can also be performed when a variable is assigned to using [cast notation](about_Variables.md).
```
